### PR TITLE
Migrating from CloudConfiguration to CloudEnvironment

### PR DESCRIPTION
**Pull request by ShihabMehboob**
Removed CloudConfiguration and replaced with CloudEnvironment.

### CODE REVIEW

File: Sources/Server/main.swift [hunk 1]
```diff
@@ -19,46 +19,34 @@ import Foundation
...
+let cloudantCredentials = cloudEnv.getCloudantCredentials(name: "MyCloudantDB")
+todos = TodoList(config: cloudantCredentials)
 let controller = TodoListController(backend: todos)
 
-let port = manager.port
+let port = cloudEnv.port
+let url = cloudEnv.url
 Log.verbose("Assigned port is \(port)")
 
-CloudFoundryDeploymentTracker(repositoryURL: "https://github.com/IBM-Swift/TodoList-CouchDB.git").track()
```
Comment (KyeMaloy97) [inline Sources/Server/main.swift:62]: Didn't seem to be any equivalent in CloudEnvironment.

**Review by rolivieri — CHANGES_REQUESTED** [review]
We need a config folder with the corresponding files in it (e.g. mappings.json, etc.). Can you add those? As reference see:

https://github.com/IBM-Swift/CloudEnvironment (REAMDE)
https://github.com/IBM-Bluemix/get-started-swift (sample app)

**Comment by rolivieri** [comment]
@ShihabMehboob Thanks for adding the config folder along with the corresponding files. A few more things to address: 1) Can you address the conflicts in the PR (you may need to rebase). 2) Have you guys tested that the application can successfully connect to the Clouding database while a) running locally, b) running as a CloudFoundry app, and c) while running as Kube app? Leveraging the CloudEnvironment library allows you to exactly do this without having to make code changes (you only need to have the files under the config folder properly configured). Let me know if there are any questions, thanks!

**Comment by rolivieri** [comment]
@ShihabMehboob Just following up on the message above: https://github.com/IBM-Swift/TodoList-CouchDB/pull/37#issuecomment-343961106. If you'd like to chat and/or have any questions, feel free to reach out on slack.

**Comment by ShihabMehboob** [comment]
@rolivieri I've migrated to CloudEnv and Armada with the latest commit, which makes use of a config folder for the CloudEnv setup, and a config.sh bash file which lets the user input commands and variables to set up the cluster, create the database, and populate it (the user gets to input credentials and their Cloudant URL).

**Comment by KyeMaloy97** [comment]
This PRs code changes have been merged into PR #40 and therefore is no longer needed. That PR will be merged into master, so this one can be closed.

**Pull request closed without merging**